src/commands/commerce/extension/register.ts

```diff
@@ -147,6 +180,9 @@ export class RegisterExtension extends SfdxCommand {
                 RegisteredExtensionName: element['DeveloperName'] as string,
                 ExtensionPointName: element['ExtensionPointName'] as string,
                 ExternalServiceProviderType: element['ExternalServiceProviderType'] as string,
+                Description: element['Description'] as string,
+                IconURI: element['IconUri'] as string,
+                IsApplication: element['IsApplication'] as string,
```

Comment:
should this be boolean?

Reply:
Oh, this is for printing the values. Is Application is an boolean element but we just turn to string when we print it. The whole getInsertedRecord just returns a string.